scania-digital-design-system/tegel
# feat(breadcrumbs): prefix change

**Describe pull-request**  
Updated prefix for breadcurmbs, also rename the component and its children to conform with convetion. 

Parent component: tds-breadcrumbs
Child component: tds-breadcrumb

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1627

**How to test**  
1. Go to storybook
2. Check in Components -> Breadcrumbs
3. Check naming of components and its style.

**Suggested test steps**
